en-us/faq.md

```diff
@@ -0,0 +1,29 @@
+# Vanilla Launcher FAQ
+
+### How to install?
+
+- There are many ways to install FO on multiple launchers. Please use this [guide](https://fabulously-optimized.gitbook.io/modpack/readme/install-instructions#minecraft-launcher-vanilla) to install FO on the launcher that you use.
+
+### So I just copy the mods, right?
+
+- No. Fabulously Optimized consists of mods, configs and a resource pack that is built to help the user naviagte through Mod Menu. If you simply copy the mods over, the settings would not be applied, and since you would have to configure the mods yourself, you might not have the proper settings to accurately replicate OptiFine.
+
+### What is the MultiMC ZIP?
+
+- MultiMC ZIP is a packaged instance for the launcher called MultiMC. It is distributed in CurseForge because MultiMC itself no longer supports downloading CurseForge Modpacks and downloads from the ZIP file support the creator of FO. This also allows for a better installation of FO for vanilla launcher users.
```

Comment:
```suggestion
- The MultiMC ZIP is a packaged instance for the launcher called MultiMC. It is distributed in CurseForge because MultiMC itself no longer supports downloading CurseForge modpacks and downloads from the ZIP file support the creator of FO. This also allows for a better installation of FO for vanilla launcher users.
```